kedro-datasets/RELEASE.md

```diff
@@ -9,6 +9,7 @@
 | `databricks.ExternalTableDataset` | A dataset for accessing external tables in Databricks. | `kedro_datasets_experimental.databricks` |
 
 ## Bug fixes and other changes
+- Fixed ibis.TableDataset to support passing arguments to `create_table`, `create_view`, and `table` via a `table_args` parameter
```

Comment:
I'd change the PR scope to `feat` and also include this under improvements; don't think it constitutes a bug, rather a lack of flexibility.

Reply:
Whoops I tried to rename the branch to resolve this and it deleted the PR 🤦  I didn't really think that through. Would it make the most sense to just open a new PR and I can add your other comment there? Sorry for the confusion, I'm a git rookie...

Reply:
Oh, you don't need to rename the branch; I mean just the PR title.

Reply:
Went ahead and moved this to the major improvements section when I resolved conflicts.